tools/reproduce_comparison/compareLinux.sh

```diff
@@ -49,7 +49,7 @@ if [ ! -r /usr/lib/jvm/jdk-${BOOTJDK_VERSION}/bin/javac ]; then
  echo Retrieving boot JDK $BOOTJDK_VERSION && mkdir -p /usr/lib/jvm && curl -L "https://api.adoptopenjdk.net/v3/binary/version/jdk-${BOOTJDK_VERSION}/linux/${NATIVE_API_ARCH}/jdk/hotspot/normal/adoptopenjdk?project=jdk" | (cd /usr/lib/jvm && tar xpzf -)
 fi
 if [ ! -r ${LOCALGCCDIR}/bin/g++-${GCCVERSION} ]; then
-  echo Retrieving gcc $GCCVERSION && curl https://ci.adoptopenjdk.net/userContent/gcc/gcc$(echo $GCCVERSION | tr -d .).`uname -m`.tar.xz | (cd /usr/local && tar xJpf -) || exit 1
+  echo Retrieving gcc $GCCVERSION && curl -L https://ci.adoptopenjdk.net/userContent/gcc/gcc$(echo $GCCVERSION | tr -d .).`uname -m`.tar.xz | (cd /usr/local && tar xJpf -) || exit 1
```

Comment:
As far as I can tell this PR doesn't do what is says? Anyway here's the change I'm guessing we need

```suggestion
  echo Retrieving gcc $GCCVERSION && curl -L https://ci.adoptium.net/userContent/gcc/gcc$(echo $GCCVERSION | tr -d .).`uname -m`.tar.xz | (cd /usr/local && tar xJpf -) || exit 1
```

Reply:
The `-L` will allow it to redirect correctly, but yes it makes sense to have the name changed so it's trying to pull from the latest URL. Suggestion accepted.